### ISSUE TITLE
modeling vehicle passengers in a way that they don't add to congestion but use real car travel time 

### ISSUE THREAD

**Issue by elyh1992** [matsim-org/matsim-code-examples#499]
Hi,

1.I have a mode called vehicle passenger in my model. If I model them as the car they would cause extra traffic on the links that are unrealistic. I would like to model them in a way that they don't add to the congestion level but use real travel time. what is your suggestion other than modeling them as teleport?

2.Also, I want to let agents change mode but I don't want the ones without a car to change their mode to the car. for this, I have added an attribute for each person that is one if they have a car and zero if they don't have a car and then I changed the 'consider car availability' variable to true in my config file.
<attributes>
			<attribute name="CarAvailability" class="java.lang.String">1</attribute>

<param name="considerCarAvailability" value="true" />

Is this the correct way for doing it?

3.and my last question is about the DRT output. Is it possible to know each DRT vehicle is having how many passengers and for example, the vehicles with more than one passenger are moving from which links? I have seen that there are some general outputs showing the total DRT occupancy time profile but I'm looking for more detailed output.

Thank you for your help in advance.

**Comment by michalmac** [comment]
> 3.and my last question is about the DRT output. Is it possible to know each DRT vehicle is having how many passengers and for example, the vehicles with more than one passenger are moving from which links? I have seen that there are some general outputs showing the total DRT occupancy time profile but I'm looking for more detailed output.

In general, we do not provide very specific and detailed insights as everyone has different questions to answer. So you would need to implement it, but you could start from `DrtVehicleOccupancyProfileCalculator` and adapt it to you needs.

**Comment by tschlenther** [comment]
Do you mean something like this
https://avoev-vsp.github.io/v/vehicle-animation/gladbeck/output-snzDrtO442l/viz-vehicles-1.yml ?

**Comment by elyh1992** [comment]
> Do you mean something like this
> https://avoev-vsp.github.io/v/vehicle-animation/gladbeck/output-snzDrtO442l/viz-vehicles-1.yml ?

Yes. This is really interesting, but even something simpler without animation that just gives the information about the occupancy of each vehicle and the links they passed is enough as well. Is it possible to use this for other locations or is it for that specific location? If you know of any code or resource that I can use as a starting point that would be great? Thank you.
